xep-0353.xml

```diff
@@ -178,38 +168,20 @@
+  <p>Instead of accepting the call, the responder might want to decline the call and tell all of her devices to stop ringing (e.g., perhaps because Romeo is getting to be a bit of a nuisance). She does this by rejecting the call on one of her devices and having that device tell all of the other devices to stop ringing by sending a &MESSAGE; stanza containing an &lt;reject/&gt; element specifying the session ID of the original &lt;propose/&gt; message to the bare JID of Romeo.</p>
+  <p>The &lt;reject/&gt; element MUST contain a &lt;reason/&gt; element as defined in &xep0166; section 7.4. The &lt;reason/&gt; element SHOULD use a condition of &lt;busy/&gt;, but implementations MAY use other conditions if deemed more appropriate (see <link url="#security">Security Considerations</link> below for details and rationale).</p>
+  <p>In Tie-Breaking scenarios it MUST also contain a &lt;tie-break/&gt; element as defined in <link url="#tie-break-1">Tie Breaking</link>.</p>
   <example caption="One of Responder's Resources Rejects the Call"><![CDATA[
-<message from='juliet@capulet.example/tablet'
-         to='juliet@capulet.example'>
-  <reject xmlns='urn:xmpp:jingle-message:0' id='a73sjjvkla37jfea'/>
-</message>
-]]></example>
-  <p>Juliet's server broadcasts this reject message to all of her available resources (as described in RFC 6121), which stop ringing:</p>
-  <example caption="Responder's Server Delivers Reject Message"><![CDATA[
-<!-- delivered to juliet@capulet.example/desktop -->
-<message from='juliet@capulet.example/tablet'
-         to='juliet@capulet.example'>
-  <reject xmlns='urn:xmpp:jingle-message:0' id='a73sjjvkla37jfea'/>
-</message>
-
-<!-- delivered to juliet@capulet.example/tablet -->
-<message from='juliet@capulet.example/tablet'
-         to='juliet@capulet.example'>
-  <reject xmlns='urn:xmpp:jingle-message:0' id='a73sjjvkla37jfea'/>
-</message>
-
-<!-- delivered to juliet@capulet.example/phone -->
-<message from='juliet@capulet.example/tablet'
-         to='juliet@capulet.example'>
-  <reject xmlns='urn:xmpp:jingle-message:0' id='a73sjjvkla37jfea'/>
-</message>
-]]></example>
-  <p>Next, the responder MAY want to decline the call explicitly, in the form of a message to the sender’s full JID containing a &lt;reject/&gt; element specifying the session ID of the original &lt;propose/&gt; message.</p>
-  <example caption="Responder Rejects the Call Explicitly to the Sender"><![CDATA[
-<message from='juliet@capulet.example/tablet'
-         to='romeo@montague.example/orchard'>
-  <reject xmlns='urn:xmpp:jingle-message:0' id='a73sjjvkla37jfea'/>
+<message from='juliet@capulet.example/phone'
+         to='romeo@montague.example'
+         type='chat'>
+  <reject xmlns='urn:xmpp:jingle-message:1' id='a73sjjvkla37jfea'>
+    <reason xmlns="urn:xmpp:jingle:1">
+      <busy/>
+      <text>Busy</text>
```

Comment:
I'm not sure we need this text element here...

Reply:
Well, the text element is optional (as per jingle xep from which the whole <reason/> element is borrowed).
I figured it could be helpful to also include optional elements in the xml examples. But I'm fine with removing the text element, too.